### DISCUSSION TITLE
Should I support AM Manager as installation method?

### DISCUSSION THREAD

**Discussion by Tobi823** [Tobi823/ffupdater#449] [Polls]
I ask because AuroraStore also support the installation with the App Manager (https://github.com/MuntashirAkon/AppManager). See.

https://gitlab.com/AuroraOSS/AuroraStore/-/blob/master/app/src/main/java/com/aurora/store/data/installer/AMInstaller.kt

https://gitlab.com/AuroraOSS/AuroraStore/-/blob/master/app/src/main/assets/installers.json#L29

**Comment by Stuikerdto** [comment]
Apk mirror installer beta seems for me the safer option, not saying the better option + works well with (ofcourse) apkmirrors apks. Aurora store is not bad, but isn't totally safe, apkmirror is olmost guaranteed safe
